hack/update-codegen.sh

```diff
@@ -33,14 +49,21 @@ chmod +x "${CODEGEN_PKG}/generate-groups.sh"
   kueue:v1beta1 \
   --go-header-file ${KUEUE_ROOT}/hack/boilerplate.go.txt
 
+# Generating applyconfiguration, clientset, informers, listers for Kueue API extensions
+"${CODEGEN_PKG}/generate-groups.sh" \
+  applyconfiguration,client,lister,informer \
+  sigs.k8s.io/kueue/client-go/api_extension \
+  sigs.k8s.io/kueue/apis \
+  api_extension:v1alpha1 \
+  --go-header-file ${KUEUE_ROOT}/hack/boilerplate.go.txt
 
 # Future releases of of code-generator add better support for out of GOPATH generation,
 # check https://github.com/kubernetes/code-generator/blob/master/examples/hack/update-codegen.sh for details,
 # for now we should just move the generated code to `client-go`.
-if  [ -d sigs.k8s.io/kueue/client-go ]
+if [ -d sigs.k8s.io/kueue/client-go ]
```

Comment:
Revert the remaining whitespace changes.

Reply:
Fixed